02 Writing Algorithms/03 Securities/99 Asset Classes/07 Futures/04 Market Hours/11 CBOT/08 Time Zone.html

```diff
@@ -1,3 +1,6 @@
 <!-- Code generated by market-hour-code-generator.py -->
 
-<p>The CBOT Future market trades in the <code>America/Chicago</code> time zone.</p>
\ No newline at end of file
+<p>The CBOT Future market trades in the following time zone(s).</p>
```

Comment:
We should only use a list if there multiple items.

If there is only 1 time zone, we should use 
> The CBOT Future market trades in the <code>America/Chicago</code> time zone.

If there is multiple time zones, we should use 
> The CBOT Future market trades in the following time zones:

and then put a list.

Note that `.` is now `:`